dash/include/dash/memory/GlobUnitMem.h

```diff
@@ -0,0 +1,470 @@
+#ifndef DASH__GLOB_UNIT_MEM_H__INCLUDED
```

Comment:
No, that's what git does when you `git mv`. Use `git log --follow` to see the full history.